app/controllers/keywords_controller.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CsvImportForm
```

Comment:
I think this form is supposed to be moved to a separated file, e.g `app/forms/csv_import_form`, no?
we don't usually combile several classes (Form, Controller) within a same file

Reply:
Fixed in f6b944240f40823b8068275f2fc8ca05590f4ce5